lib/v2/unraid/community-apps.js

```diff
@@ -0,0 +1,34 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const AppFeedUrl = 'https://raw.githubusercontent.com/Squidly271/AppFeed/master/applicationFeed.json';
+const ReturnCount = 20;
+
+module.exports = async (ctx) => {
+    const { data } = await got(AppFeedUrl);
+
+    let appList = data.applist;
+    appList = appList.map((app) => {
+        const _pubDate = app.LastUpdate ?? app.FirstSeen ?? 0;
+        return {
+            ...app,
+            _pubDate,
+        };
+    });
+    appList.sort((a, b) => b._pubDate - a._pubDate);
+    const returnedAppList = appList.slice(0, ReturnCount);
```

Comment:
Please use the common parameter `limit` from `ctx.query.limit` instead. 
docs: https://docs.rsshub.app/parameter#limit-entries